apps/charterafrica/src/components/Layout/Layout.js

```diff
@@ -8,6 +8,7 @@ import NavBar from "@/charterafrica/components/NavBar";
 
 const Main = styled("main")({
   flex: 1,
+  position: "relative",
```

Comment:
Still not sure why we need to switch this.

Reply:
Because I need to use absolute positioning somewhere in the child.

Reply:
... that's kind of the question @koechkevin . We only need to set one background image, do we really need absolute positioning? Isn't `background-position` enough?